releasenotes/notes/commutative-cancellation-preset-passmanager-c137ce516a10eae5.yaml

```
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    Added the :class`.CommutativeCancellation` pass to the ``init`` stage of the preset pass managers for
```

Oops my bad I had a typo in my suggestion:

```suggestion
    Added the :class:`.CommutativeCancellation` pass to the ``init`` stage of the preset pass managers for
```